src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -146,6 +146,22 @@ public ResponseResult rejectUser(@PathVariable("userId") Long id) {
         }
     }
 
+    @PostMapping("/reject/temp/{tempUserId}")
+    @PreAuthorize("hasAnyRole('ADMIN', 'SUPER_ADMIN')")
+    @ApiOperation(value = "Reject tempUser (for admin)")
+    public ResponseResult rejectTempUser(@PathVariable("tempUserId") Long id) {
+        try {
+            if (userService.getTempUserById(id) == null) {
+                return ResponseBuilder.fail("Temp_user with id " + id + " is not found");
```

Comment:
Lets throw an excheption EntityAlreadyExcistException here and than handle it in the controller, as @KhomchenkoAlex  already done it. Please, contact him for more info.